### ISSUE TITLE
Discussing the pipeline creation with the SD

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#3]
## User story
1. As a product owner
2. I want / need to talk to the developors about the creation of a code commit pipeline
3. So that they can get started with coding and code is checked before being added to the product

## Acceptance criteria
* Developers have decided on a realistic pipeline that is comfortable for everyone
* Developers have decided on the workload of the pipeline creation, so it can be added to the feature board 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
